product_supplierinfo_discount/views/product_supplierinfo_view.xml

```diff
@@ -1,23 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
 <odoo>
+<record model="ir.ui.view" id="product_supplierinfo_form_view">
```

Comment:
please do not change tab. (that are correct BTW). It make the PR harder to review.